### PR TITLE
Added Next Sentence Prediction 

### PR DESCRIPTION
**Pull request by EricFillion**
**Changes** 
- Completed the HappyNextSentence class. Mostly used code from the original Happy Transformer  
- Added tests under test_ns.py. Noticed that it gives a different output each time due to the model's weights being randomly initialized. 
- Included tests for when an exception is raised. We should add similar tests to other test files. 

**Questions**
- Should we rename HappyNextSentence to HappyNextSentencePrediction? 
- Should we change "predict_next_sentence' to "is_next_sentence" or something else. At first glance, people might think "predict_next_sentence" means "generate text to follow a sentence."

### CODE REVIEW

File: happytransformer/happy_next_sentence.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    def predict_next_sentence(self, sentence_a, sentence_b):
+        """
+        Determines if sentence B is likely to be a continuation after sentence
+        A.
+        :param sentence_a (string): First sentence
+        :param sentence_b (string): Second sentence to test if it comes after the first
+        :return (float): The probability that sentence_b follows sentence_a
+        """
+        if not self.__is_one_sentence(sentence_a) or not self.__is_one_sentence(sentence_b):
+            raise ValueError('Each inputted text variable for the "predict_next_sentence" method must contain a single sentence')
```
Comment (ted537) [inline happytransformer/happy_next_sentence.py:37]: I think its weird that we enforce this so strongly. If there is a false positive case, then users have no way to workaround this without modifying their input data (which could affect the transformer output).

Options:
- Remove check (trust users)
- Add option to bypass check (extra param `unsafe=False` or similar)

File: tests/test_ns.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    result = happy_ns.predict_next_sentence(
+        "How old are you?",
+        "The Eiffel Tower is in Paris."
+    )
+    assert result < 0.5
+
+
+def test_sp_sa_too_long():
+    happy_ns = HappyNextSentence()
+    with pytest.raises(Exception):
```
Comment (ted537) [inline tests/test_ns.py:28]: we can be more specific. `ValueError` could be used here

**Review by ted537 — CHANGES_REQUESTED** [review]

File: tests/test_ns.py (hunk 2 above)
Comment (EricFillion) [reply tests/test_ns.py]: No longer needed. I will be more specific next time though

File: happytransformer/happy_next_sentence.py (hunk 1 above)
Comment (EricFillion) [reply happytransformer/happy_next_sentence.py]: Good call. I also just played around with it and it seems to work well with multiple sentence inputs for both sentence_a and b. 

We'll trust the user.

**Review by ted537 — APPROVED** [review]

**Pull request merged**